server/src/main/java/org/apache/druid/indexing/overlord/supervisor/autoscaler/LagStats.java

```diff
@@ -58,6 +60,7 @@ public long getAvgLag()
    * The preferred scaling metric that supervisor may specify to be used.
    * This could be overrided by the autscaler.
    */
+  @Nullable
```

Comment:
This is not nullable (see constructor of this class). `LagStatsAutoScalerConfig.getLagAggregate()` is nullable.